### ISSUE TITLE
Display large parentheses in Windows

### ISSUE THREAD

**Issue by maprieto** [cbm755/octsympy#234]
After installing Octave-4.0.0-rc3 in Windows and using the Windows bundle (symbolic-win-py-bundle-2.2.0.zip), an unexpected warning is obtained for those outputs with large parentheses:
```
>> syms x
>> sin(x/3)
ans = (sym)

' -- converting to 'd escape sequence '\
warning: called from
    display at line 139 column 5
warning: unrecognized escape sequence '\3' -- converting to '3'
     /x
  sin|-|
     3/
```
but the right output with the ASCII encoding should be
```
ans = (sym)
     /x\
  sin|-|
    \3/
```

**Comment by cbm755** [comment]
damn, a regression from 2.1.0.  Will try to fix later this afternoon, maybe do a 2.2.1.

**Comment by cbm755** [comment]
"git bisect" is awsome.  Narrowed it down to 8ea53d7a2417c33b21b9b6d861f75 in about 2 mins...
